src/components/container/SurveyTaskList/SurveyTaskList.tsx

```diff
@@ -51,13 +52,31 @@ export default function (props: SurveyTaskListProps) {
 		}
 	};
 
+	const isSurveyBlocked = (task: SurveyTask): boolean => {
+		let blocked = false;
+
+		if (tasks && props.sequential && tasks?.length > 1) {
+			if (tasks.at(0)?.id != task.id) {
```

Comment:
Actually - is all of this checking necessary in practice?  This function is called on a specific `Task` and... you can't *have* a `Task` in any of this code unless `tasks` is non-null and there's at least one thing in it?

It seems like this function could just `return tasks![0].id === task.id`?

Reply:
Ah, yes, thanks. Didn't know about the non-null assertion operator. Pushed a change as suggested, except the equality needed to be flipped and the sequential prop still needed to be checked.